better-font-awesome.php

```diff
@@ -361,7 +361,7 @@ public function create_admin_page() {
 			<form method="post" action="options.php" id="bfa-settings-form">
 			<?php
 				// This prints out all hidden setting fields.
-				settings_fields( 'option_group' );
+				settings_fields( self::SLUG );
```

Comment:
@MickeyKay  Please note that we need to keep this change for 2 main reasons:

1) To prevent other plugins to have the same name (as I mentioned on the other pull request, this doesn't affect the saved changes of the user because when saving we were actually using better-font-awesome_options instead of option_group_options.
2) To make the nonce have the right option-group name.